django/contrib/postgres/fields/jsonb.py

```diff
@@ -118,6 +118,24 @@ class KeyTextTransform(KeyTransform):
     _output_field = TextField()
 
 
+class CoerceToText(Transform):
```

Comment:
Also, I think this can be achieved by defining `KeyTextTransform.lookup_name` and registering it as a lookup. The reason why this wasn't done in the first place is because of the possible `lookup_name` (e.g. `'as_text'`) and jsonb key collision. See [#27257](https://code.djangoproject.com/ticket/26511) for more details.